src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Bulk/submit.html.twig

```diff
@@ -29,6 +29,12 @@
         data-form-url="{{ path(action.options.submit_route, action.options.route_params) }}"
         data-form-method="{{ action.options.submit_method }}"
         data-confirm-message="{{ action.options.confirm_message }}"
+        data-close-button-label="{{ 'Close'|trans({}, 'Admin.Actions') }}"
```

Comment:
I think you don't need `data-close-button-label` unless the modal is used, so this line should in the block with the `{% if action.options.modal_options %}` condition
Besides it's a nice touch to allow modifying this label, do you think you can add this field in the `ModalOptions` php object so that it becomes configurable (you can use `confirm_button_label` as an example)